docs/core/tools/dotnet-workload-install.md

```diff
@@ -0,0 +1,104 @@
+---
+title: dotnet workload install command
+description: The 'dotnet workload install' command installs optional workloads.
+ms.date: 07/08/2021
+---
+# dotnet workload install
+
+**This article applies to:** ✔️ .NET 6 Preview SDK and later versions
+
+## Name
+
+`dotnet workload install` - Installs optional workloads.
+
+## Synopsis
+
+```dotnetcli
+dotnet workload install <WORKLOAD_ID>
+    [--add-source <SOURCE>] [--configfile <FILE>] [--disable-parallel]
+    [--download-to-cache <CACHE>] [--from-cache <CACHE>]
+    [--ignore-failed-sources] [--include-previews] [--interactive]
+    [--no-cache] [--sdk-version <VERSION>] [--skip-manifest-update]
+    [--temp-dir <PATH>] [-v|--verbosity <LEVEL>]
+
+dotnet workload install -?|-h|--help
+```
+
+## Description
+
+The `dotnet workload install` command installs an *optional workload*. Optional workloads cn be installed on top of the .NET SDK to provide support for various application types, such as [.NET MAUI](/dotnet/maui/what-is-maui) and [Blazor WebAssembly AOT](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-4/#blazor-webassembly-ahead-of-time-aot-compilation).
+
+Use [dotnet workload search](dotnet-workload-search.md) to learn what workloads are available to install.
+
+The `dotnet workload install` command copies the workloads from NuGet.org into your SDK install folder, so it needs to run elevated if that folder requires administrator write permissions.
+
+The `dotnet workload` commands operate in the context of specific SDK versions. Suppose you have both .NET 6 SDK and .NET 7 SDK  installed. The `dotnet workload` commands will provide different results depending on which SDK version you select. The commands provide the `--sdk-version` option to let you select an SDK version other than the one selected by default or by *global.json*.
```

Comment:
We should specify here that in the context of workload commands, patch versions (ex 6.0.101 and 6.0.102) will be equivalent/ give the same results whereas feature bands (ex 6.0.100 and 6.0.200) will give different results.